LondonBritishLibrary/orient/BLorient12510.xml

```diff
@@ -0,0 +1,222 @@
+                            </decoNote>
+                            <decoNote xml:id="d3" type="drawing">
+                                <locus target="#2r"/>
+                                <desc>A drawing in black ink representing 
+                                    <ref type="authFile" corresp="AT1035HolyMan">St Takla Hāymānot with three pairs of wings</ref>.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#3r"/>
+                                    <desc type="OwnershipNote">The name of the first owner is erased, the second was inscribed as
+                                        <persName ref="PRS14328ZawaldaMaryam">Zawalda Māryām</persName>.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#Ir"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="en">Pres. by <placeName ref="INS0775LWC">Wellcome Historical Medical Library</placeName> 
+                                        <date when="1961-01-04">4/1/1961</date></q>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#Ir"/>
+                                    <desc>Pen trials.</desc>  
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1"/>
+                                <decoNote type="bindingMaterial" xml:id="b2"><material key="wood"/></decoNote>
+                                <decoNote xml:id="b3" type="Boards"><material key="wood"/>Wooden boards repaired in the  
+                                    <placeName ref="INS00001BL">British Museum</placeName>.</decoNote>
+                                <decoNote xml:id="b4" type="SlipCase"><material key="leather"/>Preserved in a leather case.</decoNote>
+                                <decoNote xml:id="b5" type="Other"><term key="leafStringMark">small pieces of violet, green, and yellow yarn</term>
+                                    </decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1900" notAfter="1961">First half 20th century</origDate>
+                        </origin>
+                        <provenance>According to an inscription on the front flyleaf, it was presented by the Wellcome Historical Medical Library, 
```

Comment:
No further information on the provenance is available, i.e. in the introduction of the catalogue or so?

Reply:
As far as I can see, No. 5 was not mentioned in the Preface.